images/scripts/opensuse-tumbleweed.setup

```diff
@@ -117,12 +120,17 @@ su builder -c "sudo mock --verbose -i $(/var/lib/testvm/build-deps.sh "${ID} ${V
 
 # Prevent SSH from hanging for a long time when no external network access
 echo 'UseDNS no' >> /etc/ssh/sshd_config
+# Pam is not enabled by default and is needed for systemctl --user to work
+echo 'UsePAM yes' >> /etc/ssh/sshd_config
```

Comment:
Hmm, that worries me quite a lot -- we try to test OSes and adjust to them as they are, not how we'd like them to be. That means that when you SSH into a TW image, you don't get a user systemd and D-Bus? That will actually make things fail when you connect to TW in secondary sessions (host switcher) or with the flatpak.

Could this go into podman's test/vm.install for the time being? Then we can keep the pristine bots image as the real default config.

Reply:
There was no user systemd by default. And now that I think about it, it feels like a bug. I'll reach out the image maintainers and ask if it's intended behavior.

Reply:
So this was indeed a bug. Tumbleweed has the sshd_config in `/usr/etc/ssh/sshd_config` and creating `/etc/ssh/sshd_config` confused the system and accidentally disabled PAM

Reply:
Also now that `sshd_config` is properly set up in bots, podman's `test/vm.install` doesn't need to be modified